### ISSUE TITLE
Reopen previous opened branches in project tree

### ISSUE THREAD

**Issue by piotrzarzycki21** [prominic/Moonshine-IDE#86]
As part of that feature Moonshine will remember opened branches per project in project tree. Reopen all remembered branches in the next start of the project or IDE.

**Comment by piotrzarzycki21** [comment]
Reopen cause I forgot to implement that reset settings clean up saved branches.

**Comment by rat-moonshine** [comment]
When I tested this I found something a little odd:
- When reopened Moonshine I didn't seen the project panel opened automatically (actually that was what I expected by definition); even opening the project panel from top menu, it didn't had any last project references opened
- When I tried to reopen a previously opened project - it opened suddenly with all the folder leaf opened last time - which in my opinion a little odd and 'did not expected' situation to me

Basically, I was expecting this behavior (by nature), to open up all the last time opened project automatically with their last tree state in the project panel. This will basically push Moonshine to show the project panel everytime when restarts (unlike the way is now).

**Comment by piotrzarzycki21** [comment]
To me your expectations above is a different feature, but connected to that one. Please raise separate Issue and I will add it to my To Do list.

**Comment by JoelProminic** [comment]
The branch states were preserved properly in my tests.
